01-node-tutorial/answers/12-http.js

```diff
@@ -0,0 +1,19 @@
+const http = require('http')
+
+const server = http.createServer((req, res) => {
+  if (req.url === '/') {
+    res.end('Welcome to our home page')
+  } else if (req.url === '/about') {
+    res.end('Here is our short history')
+  } else if (req.url === '/contact') {
+    res.end('Contact information')
+  } else {
+    res.end(`
+    <h1>Oops!</h1>
+    <p>We can't seem to find the page you are looking for</p>
+    <a href="/">back home</a>
```

Comment:
Extra challenge (optional): change the error message to say something fun. Use some different html elements, and maybe add CSS to the page as well. You don't need to create a new file, you can just put everything inline:

```javascript
res.end(`
    <style>
        p {
            color: red
        }

        a:hover {
            color: orange;
        }
    </style>

    <h1>Oops!</h1>
    <p>We can't seem to find the page you are looking for!</p>
    <a href="/">back home</a>
`)
 
```